src/main/resources/db/migration/all/20241128163133__application_controller_and_linked_services_in_cas2bail.sql

```diff
@@ -0,0 +1,157 @@
+
+CREATE TABLE cas_2_bail_application_json_schemas
+(
+    json_schema_id UUID NOT NULL,
+    CONSTRAINT pk_cas_2_bail_application_json_schemas PRIMARY KEY (json_schema_id)
+);
+
+CREATE TABLE cas_2_bail_application_notes
+(
+    id                          UUID NOT NULL,
+    application_id              UUID,
+    created_at                  TIMESTAMP WITHOUT TIME ZONE,
+    body                        VARCHAR(255),
```

Comment:
can you please revise this file to match the Cas2 database types on equivalent tables? If you think they're wrong give me a shout and we can discuss